### ISSUE TITLE
add `app/levureBuildStandaloneTemp*.livecode` to .gitignore?

### ISSUE THREAD

**Issue by macMikey** [trevordevore/levure#176]
After one starts doing builds, a number of levureBuildStandaloneTemp stacks end up being proposed for the repo, which I think is incorrect.
Side note: building the app also changes the standalone.livecode file, which I think adds unnecessary noise to the project history. It seems that those changes are primarily the version/build numbering and the inclusion of the standalonetemp stacks.
There are some changes to standalone.livecode that should be tracked, namely things like platform build prefs and inclusions, but the other updates shouldn't be.

**Comment by trevordevore** [comment]
@macMikey There shouldn't be any `levureBuildStandaloneTemp` stacks left over after a build is done. Are you using git on the command line or a GUI? I've seen GUIs pick up the temp stack during the build and display it as needing to be committed, but then not remove it after the temp file is deleted. Can you confirm that the temp file actually exists after the packaging operation is done?

As for standalone.livecode, I'm not sure what the best thing to do is. I always discard the changes to standalone.livecode, but the file is actually being changed to include the latest version number in the custom props used for the LiveCode SE builder. The inclusion of the temp stack shouldn't be saved with it though. Have you confirmed that the temp stacks are still part of the saved `standalone.livecode` stack after the packaging operation finishes?

**Comment by macMikey** [comment]
I just did two `levurePackageApplication "release"`. After the first one, I have two levureBuildStandaloneTemp stacks sitting in the app directory. I quit LC, deleted those stacks, built again, and no temp stacks this time.
Also, when I looked at my *standalone.livecode*, there were several levure temp files listed in the *copy files* pane. Removing those from the list and building again did not cause any to reappear.
I'll keep fiddling to see if I can come up with a good, reliable recipe.
